docs/sdk/src/reference_docs/frame_currency.rs

```diff
@@ -1,8 +1,55 @@
-//! FRAME Currency Abstractions and Traits
+//! # FRAME Currency Abstractions and Traits
 //!
-//! Notes:
+//! Currency related traits in FRAME provide standardized interfaces for implementing various
+//! currency functionalities. The fundamental reason to have these traits is to allow pallets to
+//! utilize a token without relying on the implementation detail. Or else we could have a perfectly
```

Comment:
> Or else we could have a perfectly fine runtime without these traits.

I don't quite get what this sentence means to say. Could you elaborate/rephrase?

Reply:
It should be removed as it is redundant, what I meant is that you don't need these traits if your runtime doesn't require a currency